packages/api-client/src/main/team/team/NewTeamData.ts

```diff
@@ -20,8 +20,13 @@
 import {MemberData} from '../member/MemberData';
 
 export interface NewTeamData {
+  /** User binding team */
+  binding?: boolean;
```

Comment:
What does it mean to be a "User binding team"?

Reply:
If it's not "binding", it's a legacy team.

Reply:
It stems from the initial idea of teams. There you were able to bind an account to multiple teams.